scripts/add-javascript-rules.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Absolutely, most likely it will be only the dotnet team to run it. This should work fine in Git Bash as is, but if that's not convenient for you then I can try to improve it the next time I work in Windows.